component-catalog-connectors/airflow-example-components-connector/setup.py

```diff
@@ -38,7 +38,7 @@
     license="Apache License Version 2.0",
     packages=find_packages(),
     install_requires=[
-        'elyra==3.3.0.dev0'
+        'elyra>3.2'
```

Comment:
If elyra has a dependency on this, will this trigger a circular dependency?  I suppose for elyra, the plan is to use an `extras_require` to pull in the examples: `pip install elyra[examples]` or `pip install elyra[examples-kfp]` - so we're probably okay.

Reply:
I removed the dependency